### PR TITLE
Fix for Client Index XX Error.

### PR DESCRIPTION
**Pull request by vexx-sm**
Started at #33

### CODE REVIEW

File: scripting/tf2itemplugin_weapons.sp [hunk 1]
```diff
@@ -9,273 +17,260 @@
 #pragma newdecls required
 
 #define PLUGIN_VERSION "4.0.0"
```
Comment (punteroo) [inline scripting/tf2itemplugin_weapons.sp]: Adjust version number to 4.0.1

File: scripting/tf2itemplugin_weapons.sp [hunk 2]
```diff
@@ -9,273 +17,260 @@
...
 
 public Plugin myinfo =
 {
-	name		= "TF2 Item Plugins - Weapons Manager",
-	author		= "Lucas 'punteroo' Maza",
-	description = "Customize your weapons and manage your server inventory.",
-	version		= PLUGIN_VERSION,
-	url			= "https://github.com/punteroo/TF2-Item-Plugins"
+    name        = "TF2 Item Plugins - Weapons Manager",
+    author      = "Lucas 'punteroo' Maza",
```
Comment (punteroo) [inline scripting/tf2itemplugin_weapons.sp:25]: You can add yourself as a co-author such as `& vexx-sm` if you like.

File: scripting/tf2itemplugin_cosmetics.sp [hunk 3]
```diff
@@ -9,15 +9,15 @@
 #pragma newdecls required
 
 #define PLUGIN_VERSION "4.0.0"
```
Comment (punteroo) [inline scripting/tf2itemplugin_cosmetics.sp]: Adjust version number to 4.0.1

File: scripting/tf2itemplugin_cosmetics.sp [hunk 4]
```diff
@@ -9,15 +9,15 @@
...
 
 public Plugin myinfo =
 {
-	name		= "TF2 Item Plugins - Cosmetics Manager",
-	author		= "Lucas 'punteroo' Maza",
-	description = "Customize your cosmetic items and manage your server inventory.",
-	version		= PLUGIN_VERSION,
-	url			= "https://github.com/punteroo/TF2-Item-Plugins"
+    name        = "TF2 Item Plugins - Cosmetics Manager",
+    author      = "Lucas 'punteroo' Maza",
```
Comment (punteroo) [inline scripting/tf2itemplugin_cosmetics.sp:17]: You can add yourself as a co-author such as `& vexx-sm` if you like.

**Review by punteroo — CHANGES_REQUESTED** [review]
Please adjust version numbers (and add yourself as co-author if you like) and I'll approve your pull.

File: scripting/tf2itemplugin_weapons.sp (hunk 2 above)
Comment (vexx-sm) [reply scripting/tf2itemplugin_weapons.sp:25]: That wouldn't be necessary, Just helping out :)

**Review by punteroo — APPROVED** [review]
Looks good!

**Pull request merged**